src/ClientHandle.h

```diff
@@ -377,6 +377,7 @@ class cClientHandle  // tolua_export
 
 
 	void HandlePluginMessage    (const AString & a_Channel, ContiguousByteBufferView a_Message);
+	void HandleResourcePack     (UInt8 a_Status);
```

Comment:
I debated between naming this function `HandleResourcePack` and `HandleResourcePackStatus`.
I chose the former because:
1. naming it the latter would mean I would have to add spaces to the rest of the lines to realign the function declarations
2. In later versions of the protocol on the wiki, this packet gets renamed to 'Resource Pack (serverbound)'